### PR TITLE
Optimize imports & add spaces

### PR DESCRIPTION
**Pull request by AlexProgrammerDE**

### CODE REVIEW

File: protocol/src/main/java/org/geysermc/mcprotocollib/network/helper/TransportHelper.java [hunk 1]
```diff
@@ -10,7 +10,8 @@ public enum TransportMethod {
     }
 
     public static TransportMethod determineTransportMethod() {
-        if (isClassAvailable("io.netty.incubator.channel.uring.IOUring") && IOUring.isAvailable()) return TransportMethod.IO_URING;
+        if (isClassAvailable("io.netty.incubator.channel.uring.IOUring") && IOUring.isAvailable())
+            return TransportMethod.IO_URING;
```
Comment (basaigh) [inline protocol/src/main/java/org/geysermc/mcprotocollib/network/helper/TransportHelper.java]: Please revert.

**Review by onebeastchris — APPROVED** [review]

File: protocol/src/main/java/org/geysermc/mcprotocollib/protocol/ServerListener.java [hunk 2]
```diff
@@ -103,7 +103,8 @@ public void packetReceived(Session session, Packet packet) {
                             session.disconnect("Outdated client! Please use " + protocol.getCodec().getMinecraftVersion() + ".");
                         }
                     }
-                    default -> throw new UnsupportedOperationException("Invalid client intent: " + intentionPacket.getIntent());
+                    default ->
+                            throw new UnsupportedOperationException("Invalid client intent: " + intentionPacket.getIntent());
```
Comment (basaigh) [inline protocol/src/main/java/org/geysermc/mcprotocollib/protocol/ServerListener.java]: Please revert.

File: protocol/src/main/java/org/geysermc/mcprotocollib/protocol/data/game/item/component/DataComponentType.java [hunk 3]
```diff
@@ -162,7 +162,8 @@ private static Reader<Unit> unitReader() {
     }
 
     private static Writer<Unit> unitWriter() {
-        return (helper, output, value) -> {};
+        return (helper, output, value) -> {
+        };
```
Comment (basaigh) [inline protocol/src/main/java/org/geysermc/mcprotocollib/protocol/data/game/item/component/DataComponentType.java:166]: Please revert.

File: protocol/src/main/java/org/geysermc/mcprotocollib/protocol/codec/NbtComponentSerializer.java [hunk 4]
```diff
@@ -26,21 +26,21 @@
 public class NbtComponentSerializer {
 
     private static final Set<String> BOOLEAN_TYPES = Set.of(
-        "interpret",
-        "bold",
-        "italic",
-        "underlined",
-        "strikethrough",
-        "obfuscated"
+            "interpret",
```
Comment (basaigh) [inline protocol/src/main/java/org/geysermc/mcprotocollib/protocol/codec/NbtComponentSerializer.java:28]: Please revert this and below.

**Review by basaigh — CHANGES_REQUESTED** [review]

**Review by onebeastchris — APPROVED** [review]

**Review by basaigh — APPROVED** [review]

**Pull request merged**